buildtest/buildsystem/checks.py

```diff
@@ -169,12 +169,12 @@ def is_symlink_check(builder):
 
         if is_symlink(sym_link):
```

Comment:
can you change the `sym_link` variable to `fname` or `filename` when u are going through the for loop. It will help with code readability. There is a variable `sym_link` then there is a method `is_symlink` and then we also have this method `is_symlink_check` so it is already getting quite confusing so best to use meaningful variable names.

Reply:
Okay done